index.html

```diff
@@ -57,6 +57,266 @@ <h1 id="primary-text">
                 <img src="icons/union.svg" />
             </button>
         </div>
+
+        <div id="recent-works">
+            <p class="text">My Recent Works</p>
+        
+            <div id="arrow-ctn">
+                <img src="icons/indicator.svg" />
+            </div>
+        </div>
+        
+        <section id="works">
+            <div class="work-card">
+                <div class="top-area">
+        
+                </div>
+                <div class="bottom-area">
+                    <div class="top">
+                        <p class="text">
+                            Multi-post stories<br>
+                            Gain+Glory
+                        </p>
+                    </div>
+                    <div class="tech">
+                        <div class="tech-card">
+                            <p>Ruby on Rails</p>
+                        </div>
+                        <div class="tech-card">
+                            <p>CSS</p>
+                        </div>
+                        <div class="tech-card">
+                            <p>JavaScript</p>
+                        </div>
+                        <div class="tech-card">
+                            <p>HTML</p>
+                        </div>
+                    </div>
+                    <div class="bottom">
+                        <div class="see-btn">
+                            <p>See Project</p>
+                        </div>
```

Comment:
- [ ] Kindly make this a `button` or a `a` tag. This is a  call to action that requires users to click in order to engage. So, a `p` does not really fits here even though it renders :muscle:
- [ ] In like manner please apply this to all the button on your page - (`See project`, `Get resume`, and any other) :wink: